README.md

```diff
@@ -52,6 +52,8 @@ The Marquez [HTTP API](https://marquezproject.github.io/marquez/openapi.html) li
 
 > **Note:** By default, the HTTP API does not require any form of authentication or authorization.
 
+> **Note:** Port 5000 is now reserved for MacOS. If running locally on MacOS, you can run `./docker/up.sh --api-port 9000` to configure the API to listen on port 9000 instead. Keep in mind that you will need to update the URLs below with the appropriate port number.
```

Comment:
Thanks for adding calcification around the port! I was wondering, should we add the note closer to where the `./docker/up.sh` script is run? I.e. right under the [Quickstart](https://github.com/MarquezProject/marquez#quickstart) section?

Reply:
Yep, that makes way more sense! Long day :)